src/includes/constants.php

```diff
@@ -15,9 +15,9 @@
  * - BASE_URL: Full URL to the webroot, e.g. "http://my-host.com/my-webroot" (no trailing slash).
  * - BASE_PATH: Absolute path to the webroot, e.g. "/var/www/my-webroot" (no trailing slash).
  *   See Director::baseFolder(). Can be overwritten by Config::modify()->set(Director::class, 'alternate_base_folder', ).
- * - TEMP_PATH: Absolute path to temporary folder, used as base for $TEMP_FOLDER. Example: "/var/tmp"
- * - TEMP_FOLDER: folder name for manifest and template caches. Example: "silverstripe-cache-php7.2"
- *   See getTempFolder(). No trailing slash.
+ * - TEMP_PATH: Path to temporary folder for manifest and template caches. May be relative to project root or an
```

Comment:
Should we mention SS_TEMP_PATH here as well? (possibly not?)

Reply:
Have added a small note about the new environment variable here